### PR TITLE
Add experimental HTTP attributes to HTTP spans

### PR DESCRIPTION
**Pull request by lmolkova**
## Changes

Adds HTTP size attributes to HTTP semconv

## Merge requirement checklist

* [x] [CONTRIBUTING.md](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md) guidelines followed.
* [x] Change log entry added, according to the guidelines in [When to add a changelog entry](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#when-to-add-a-changelog-entry).
  * If your PR does not need a change log, start the PR title with `[chore]`
* ~~[schema-next.yaml](https://github.com/open-telemetry/semantic-conventions/blob/main/schema-next.yaml) updated with changes to existing conventions.~~

### CODE REVIEW

File: .chloggen/988.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+change_type: enhancement
+
+component: http
+
+note: List experimental HTTP attributes applicable to HTTP client and server spans.
+
+issues: [988]
```
Comment (trask) [inline .chloggen/988.yaml:7]:
```suggestion
issues: [989]
```

File: docs/http/http-spans.md [hunk 2]
```diff
@@ -17,6 +17,7 @@ and various HTTP versions like 1.1, 2 and SPDY.
 - [Name](#name)
 - [Status](#status)
 - [HTTP client](#http-client)
+  - [HTTP Client Experimental Attributes](#http-client-experimental-attributes)
```
Comment (trask) [inline docs/http/http-spans.md:20]: (to match sentence case of other headings)
```suggestion
  - [HTTP client experimental attributes](#http-client-experimental-attributes)
```

File: docs/http/http-spans.md [hunk 3]
```diff
@@ -240,6 +242,21 @@ The following attributes can be important for making sampling decisions and SHOU
 | `unix` | Unix domain socket | ![Stable](https://img.shields.io/badge/-stable-lightgreen) |
 <!-- endsemconv -->
 
+### HTTP Client Experimental Attributes
+
+**Status**: [Experimental][DocumentStatus]
+
+Instrumentations MAY allow users to enable additional experimental attributes. The following attributes are define for HTTP Client spans
```
Comment (trask) [inline docs/http/http-spans.md:249]: (maybe not needed, and then don't need to decide whether to end in `.` or `:` 😂)
```suggestion
Instrumentations MAY allow users to enable additional experimental attributes.
```

File: docs/http/http-spans.md [hunk 4]
```diff
@@ -458,6 +475,21 @@ The following attributes can be important for making sampling decisions and SHOU
 
 `http.route` MUST be provided at span creation time if and only if it's already available. If it becomes available after span starts, instrumentation MUST populate it anytime before span ends.
 
+### HTTP Server Experimental Attributes
+
+**Status**: [Experimental][DocumentStatus]
+
+Instrumentations MAY allow users to enable additional experimental attributes. The following attributes are define for HTTP Server spans
```
Comment (trask) [inline docs/http/http-spans.md:482]: (similar to above)
```suggestion
Instrumentations MAY allow users to enable additional experimental attributes.
```

**Review by trask — APPROVED** [review]

**Review by joaopgrassi — APPROVED** [review]

**Pull request merged**